cli/src/commands/uninstall.rs

```diff
@@ -0,0 +1,141 @@
+    }
+
+    if let Err(err) = Shell::Zsh.cleanup() {
+        print_user_warning!("ZSH config error: {}", err);
+    }
+
+    Ok(CommandValue::Void)
+}
+
+/// Remove the entire `~/.phylum` directory.
+fn purge() -> Result<()> {
+    let path = home_dir()?.join(".phylum");
+    match fs::remove_dir_all(path) {
+        Ok(()) => print_user_success!("Successfully removed ~/.phylum."),
+        Err(err) => {
+            print_user_warning!("Could not remove phylum config directory: {}", err);
+        }
+    }
+
+    Ok(())
+}
+
+/// Remove files created by `install.sh`.
+fn remove_installed_files() -> Result<()> {
+    let phylum_dir = home_dir()?.join(".phylum");
+    if let Err(err) = fs::remove_dir_all(phylum_dir.join("completions")) {
+        print_user_warning!("Could not remove completions directory: {}", err);
+    }
+
+    if let Err(err) = fs::remove_file(phylum_dir.join("phylum")) {
+        print_user_warning!("Could not remove phylum executable: {}", err);
+    }
+
+    // TODO: Should we print success if there were errors?
+    print_user_success!("Successfully removed installer files.");
+
+    Ok(())
+}
+
+/// Supported shells.
+#[derive(Debug)]
+enum Shell {
+    Bash,
+    Zsh,
+}
+
+impl Shell {
+    /// Get the shell's config path.
+    fn rc_path(&self) -> Result<PathBuf> {
+        match self {
+            Self::Bash => Ok(home_dir()?.join(".bashrc")),
+            Self::Zsh => Ok(home_dir()?.join(".zshrc")),
+        }
+    }
+
+    /// Lines added to the config by the phylum installer.
+    fn phylum_config(&self) -> &str {
+        match self {
+            Self::Bash => BASH_TEXT,
+            Self::Zsh => ZSH_TEXT,
+        }
+    }
+
+    /// Remove all lines from the shell config which were added by the installer.
+    fn cleanup(&self) -> Result<()> {
+        let rc_path = self.rc_path()?;
+        let mut rc_content = fs::read_to_string(&rc_path)?;
+
+        // If the installer's config is present in full, remove it.
+        let phylum_config = self.phylum_config();
+        if !rc_content.contains(phylum_config) {
+            return Err(anyhow!(
+                "Could not find config in {:?} shell. \
+                Manual removal might be necessary.",
+                self
+            ));
+        }
+        rc_content = rc_content.replace(phylum_config, "");
+
+        // Write to tempfile on same mountpoint to avoid accidental corruption.
+        let rc_dir = rc_path
+            .parent()
+            .ok_or_else(|| anyhow!("Shell file has no parent directory"))?;
+        let mut tmpfile = NamedTempFile::new_in(rc_dir)?;
+        tmpfile.write_all(rc_content.as_bytes())?;
+
+        // Swap the tempfile into place.
+        tmpfile.persist(self.rc_path()?)?;
+
+        print_user_success!("Removed entries from {:?} config.", self);
+
+        Ok(())
+    }
+}
+
+/// Get the user's home directory.
+fn home_dir() -> Result<PathBuf> {
+    dirs::home_dir().ok_or(anyhow!("Unable to find home directory"))
```

Comment:
In [config.rs](https://github.com/phylum-dev/cli/blob/c5eb1010f66a703f04cc31b320b4fe0d5d468bef/cli/src/config.rs#L161), we're using `home::home_dir()`, but here we use `dirs::home_dir()`. Can we consolidate and use the same one in both places?